flink-runtime-web/web-dashboard/src/app/interfaces/configuration.ts

```diff
@@ -22,4 +22,7 @@ export interface ConfigurationInterface {
   'timezone-offset': number;
   'flink-version': string;
   'flink-revision': string;
+  features?: {
```

Comment:
why are we checking this for null?

Reply:
Users may use this web for flink 1.9.0 and below, some company depends on flink ui to build there own flink platform.